minian/utilities.py

```diff
@@ -113,7 +113,16 @@ def load_videos(vpath,
             "No data with pattern {}"
             " found in the specified folder {}".format(pattern, vpath))
     print("loading {} videos in folder {}".format(len(vlist), vpath))
-    varr_list = [load_avi_lazy(v) for v in vlist]
+
+    file_extension = os.path.splitext(vlist[0])[1]
+    if file_extension == '.avi':
+        movie_load_func = load_avi_lazy
+    elif file_extension == '.tif':
+        movie_load_func = load_tif_lazy
+    else:
+        raise ValueError('Extension not supported.')
+
+    varr_list = [movie_load_func(v) for v in vlist]
```

Comment:
I would be careful to follow this exact same flow for tif here. When I wrote this `v` was a single video, which contains 1000 frames, `load_avi_lazy` was designed to do a lazy loading in that case, then they are concatenated here. I see you almost copied that behavior for `load_tif_lazy`, which would make sense for several tif stacks that should be concatenated together, but not when each frame is a single tif file. I think it would produce significant overhead in that case or no longer doing a lazy load at all (since there is a `compute` of the first frame in `load_tif_lazy`). Have you tested this with a folder of single-frame tifs?